app/views/layouts/application.html.erb

```diff
@@ -1,5 +1,6 @@
 <!DOCTYPE html>
-<html lang="<%= FastGettext.locale %>">
+<html lang="<%= FastGettext.locale %>"   style="position: relative;
```

Comment:
Oops. I put some of them in `dmproadmap.scss` but some got away. I'll fix and re-push.